cmd/krel/cmd/gcbmgr.go

```diff
@@ -186,63 +230,66 @@ func runGcbmgr() error {
 		logrus.Infof("%s: %s", k, v)
 	}
 
+	var jobType string
 	switch {
+	// TODO: Consider a '--validate' flag to validate the GCB config without submitting
 	case gcbmgrOpts.stage:
-		return submitStage(toolRoot, gcbSubs)
+		jobType = "stage"
 	case gcbmgrOpts.release:
-		return submitRelease()
+		jobType = "release"
+		buildOpts.DiskSize = "100"
 	default:
 		return listJobs()
 	}
-}
 
-func submitStage(toolRoot string, substitutions map[string]string) error {
-	logrus.Infof("Submitting a stage to GCB")
-
-	buildOpts.CloudbuildFile = filepath.Join(toolRoot, "gcb/stage/cloudbuild.yaml")
+	buildOpts.ConfigDir = filepath.Join(toolRoot, "gcb", jobType)
+	prepareBuildErr := build.PrepareBuilds(buildOpts)
+	if prepareBuildErr != nil {
+		return prepareBuildErr
+	}
```

Comment:
why not:
```go
if err := build.PrepareBuilds(buildOpts); err != nil {
	return err
}
```
?

Reply:
I have a slight readability preference with the current form.